packages/sdk/src/transactions/PSBTBuilder.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```

Comment:
i think you want to remove the no-console eslint and all the console logs as well

Reply:
oops this wasnt meant to be committed, removing